scipy/sparse/linalg/tests/test_special_sparse_arrays.py

```diff
@@ -129,3 +153,7 @@ def test_dot(self, grid_shape, bc, dtype):
             y = lap.dot(x.astype(dtype))
             assert x.shape == y.shape
             assert y.dtype == dtype
+
+def test_boundary_conditions_value_error():
+    with pytest.raises(ValueError, match="Unknown value 'robin'"):
+        LaplacianNd(grid_shape=(6, ), boundary_conditions='robin')
```

Comment:
Lets put this test with the others
```suggestion
    def test_boundary_conditions_value_error(self):
        with pytest.raises(ValueError, match="Unknown value 'robin'"):
            LaplacianNd(grid_shape=(6, ), boundary_conditions='robin')
```